raiden-cli/src/config.json

```diff
@@ -4,6 +4,20 @@
   "autoSettle": true,
   "autoUDCWithdraw": false,
   "caps": {
-    "Receive": 1
+    "Receive": 1,
+    "Mediate": 1,
+  },
+  "mediationFees": {
+    "0x0000000000000000000000000000000000000000": {
+      "flat": 0, 
+      "prop": 4000,
+      "inbalance": 3000,
+    },
+    "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2": {
+      "flat": 10000000000,
```

Comment:
Notice that setting an object containing only e.g. `flat` will shadow the whole default `AddressZero` config, meaning these tokens will have no config for the other models. Is this intended?

Reply:
See the discussion in the review comment conversation above.

Reply:
✔️